### DISCUSSION TITLE
fluent-bit collects the entire file

### DISCUSSION THREAD

**Discussion by 666MrFang** [fluent/fluent-bit#6094] [Q&A]
How fluent-bit collects the entire file, such as collecting a binary, a gz format compressed package？what input plugin can i use

**Comment by edsiper** [comment]
input plugins does not support binary file or a .gz file, Tail input plugin read text plain files only.

**Reply by 666MrFang** [reply]
Sir, ask the next two questions: 1. Does the forward plugin in the input plugin support binary data streams? 2. From my test, the http plugin  in the input plugin does not support reporting files. Can the content-type only be application/json?Can't be multipart/form-data?
